### PR TITLE
✨ Added Int.up(to:) helper method

### PR DESCRIPTION
**Pull request by cHaLkdusT**
Iterates from this number up to the given number, inclusive, incrementing by one each time.
### Sample Usage
`2.up(to: 5) { print($0) }`

## Checklist

- [x] I checked the [**Contributing Guidelines**](https://github.com/SwifterSwift/SwifterSwift/blob/master/CONTRIBUTING.md) before creating this request.
- [x] New extensions are written in Swift 5.0.
- [x] New extensions support iOS 8.0+ / tvOS 9.0+ / macOS 10.10+ / watchOS 2.0+, or use `@available` if not.
- [x] I have added tests for new extensions, and they passed.
- [x] All extensions have a **clear** comments explaining their functionality, all parameters and return type in English.
- [x] All extensions are declared as **public**.
- [x] I have added a [changelog](https://github.com/SwifterSwift/SwifterSwift/blob/master/CHANGELOG_GUIDELINES.md) entry describing my changes.

### CODE REVIEW

**Comment by LucianoPAlmeida** [comment]
Hey @cHaLkdusT :) Thank's for taking the time to send this PR, but actually, there is already a very simple and nice way to do this on the stdlib using range types like `(2..<5).forEach { print($0) }`

**Comment by cHaLkdusT** [comment]
Applied all suggested changes!

**Comment by LucianoPAlmeida** [comment]
I'm still not sure about this one, I think  `(2..<5).forEach { print($0) }` is more readable and simple enough for not worth the abstraction. So my feeling is that it is a bit over-abstraction.
Let's see what the others think cc @omaralbeik @SD10 @BennX

File: Sources/SwifterSwift/SwiftStdlib/ComparableExtensions.swift [hunk 1]
```diff
@@ -41,3 +41,21 @@ public extension Comparable {
...
+
+public extension Comparable where Self: Strideable, Self.Stride: SignedInteger {
+    
+    /// Iterates from this number up to the given number, inclusive,
+    /// incrementing by one each time.
+    ///
+    /// - Parameters:
+    ///   - to: another Int to go up to
+    ///   - callback: The closure to call
+    func up(to: Self, callback: (_ current: Self) -> Void) {
```
Comment (chulbert) [inline Sources/SwifterSwift/SwiftStdlib/ComparableExtensions.swift]: I might recommend `up(through:, callback:)` to better convey the inclusivity of a closed range.

File: Sources/SwifterSwift/SwiftStdlib/ComparableExtensions.swift [hunk 2]
```diff
@@ -41,3 +41,21 @@ public extension Comparable {
...
+        guard to >= self else {
+            fatalError("The argument \(to) to up(to:) cannot be less than the value \(self) it's called on.")
```
Comment (chulbert) [inline Sources/SwifterSwift/SwiftStdlib/ComparableExtensions.swift]: This guard is unnecessary.  The same outcome will occur when the closed range below is created.

**Comment by LucianoPAlmeida** [comment]
Just cleaning up those old PR's. 
@cHaLkdusT Closing this one due to inactivity on the discussion, feel free to reopen if you like it :))

**Pull request closed without merging**